index.js

```diff
@@ -1,10 +1,23 @@
 const { EventEmitter } = require('events');
-const Wallet = require('ethereumjs-wallet').default;
 const ethUtil = require('ethereumjs-util');
+const randomBytes = require('randombytes');
 
 const type = 'Simple Key Pair';
 const sigUtil = require('eth-sig-util');
 
+function generateKey() {
+  const privateKey = randomBytes(32);
+  // I don't think this is possible, but this validation was here previously,
+  // so it has been preserved just in case.
+  // istanbul ignore next
+  if (!ethUtil.isValidPrivate(privateKey)) {
```

Comment:
validation happens down a few deps
https://github.com/cryptocoinjs/secp256k1-node/blob/024fbdad3fb64499e8db8b35246c2f0a36afa8c8/lib/elliptic.js#L76
the `n` value is very big
https://github.com/indutny/elliptic/blob/43ac7f230069bd1575e1e4a58394a512303ba803/lib/elliptic/curves.js#L182

it is possible but improbable to fail the validation